Frameworks/OakTabBarView/src/OakTabItemView.mm

```diff
@@ -151,8 +200,35 @@ - (id)init
 		_inactiveTabTextStyles = _activeTabTextStyles.mutableCopy;
 		_inactiveTabTextStyles[NSForegroundColorAttributeName] = [NSColor colorWithCalibratedWhite:0.5 alpha:1];
 
+		if([NSProcessInfo instancesRespondToSelector:@selector(isOperatingSystemAtLeastVersion:)] && [[NSProcessInfo processInfo] isOperatingSystemAtLeastVersion:{ 10, 14, 0 }])
+		{
+			_activeTabTextStyles = @{
+				NSParagraphStyleAttributeName  : parStyle,
+				NSFontAttributeName            : [NSFont systemFontOfSize:11],
+				NSForegroundColorAttributeName : [NSColor secondaryLabelColor],
+			}.mutableCopy;
+
+			_inactiveTabTextStyles = _activeTabTextStyles.mutableCopy;
+			_inactiveTabTextStyles[NSForegroundColorAttributeName] = [NSColor tertiaryLabelColor];
+			_selectedTabTextStyles = _activeTabTextStyles.mutableCopy;
+			_selectedTabTextStyles[NSForegroundColorAttributeName] = [NSColor labelColor];
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunguarded-availability"
+			NSAppearanceName appearanceName = [[NSApplication sharedApplication].effectiveAppearance bestMatchFromAppearancesWithNames:@[ NSAppearanceNameAqua, NSAppearanceNameDarkAqua]];
+			if([appearanceName isEqualToString:NSAppearanceNameDarkAqua])
+				_images = [self mojaveDarkImages];
+			else
+				_images = [self mojaveLightImages];
+#pragma clang diagnostic pop
+
```

Comment:
This section should really be enclosed in an `@available` check, but I was trying to make sure this would still compile with Xcode 7.